docs/source/reference-core.rst

```diff
@@ -837,104 +867,6 @@ The nursery API
    See :meth:`~Nursery.start`.
 
 
-Working with :exc:`MultiError`\s
```

Comment:
There's nothing wrong with it, but it does feel like a downgrade when the previous set of docs had multiple examples, describing more edge cases, exception chaining, output, etc.

Reply:
Ok, I'll restore and update the docs.

Reply:
I took the time to evaluate the examples from the removed section, but since the catching API in the backport works in such a different way, I'm not sure what there is to document. The backport doesn't provide any functionality that the `except*` syntax doesn't. There's no exception conversion, and to reraise exceptions, you just use `raise` as usual. That said, I've amended the documentation to at least show how to iterate over the exceptions in the group.